prow/config/jobs.go

```diff
@@ -153,6 +153,13 @@ type Postsubmit struct {
 
 	Brancher
 
+	// Context is the name of the GitHub status context for the job.
+	Context string `json:"context"`
+
+	// TODO(krzyzacy): OPT out by default for now? Do we want to opt everyone in?
```

Comment:
I think you mean opt-in (aka opt-into report, defaulting to off). That 👍

Reply:
Default is false which is not reporting right? (Here is Report oppose to SkipReport for presubmits)

Currently it's guarded by https://github.com/kubernetes/test-infra/blob/master/prow/report/report.go#L110-L112, it will report the same way presubmit does (Comment to the PR, also create a status to the commit)

Reply:
Right, I'm just commenting that you have the meanings of opt-in and opt-out reversed: https://en.wiktionary.org/wiki/opt-in


Defaulting to report: false, with an option to report: true, is an opt-in pattern. (like --confirm)

By contrast, an option to print noisy debug messages, with an option to `--silent` is an opt-out of logging option.

Reply:
ehhh... so `opt-out` is not the same as `not opt-in`? fun...

Reply:
ohhh `opt-in` is just an adj. that describes the system...